examples/bigquery-snowflake-tables-migration-utility/BQ_Table_Creator/bq_table_creator.py

```diff
@@ -0,0 +1,418 @@
+# Copyright 2022 Google LLC
+
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+
+#        http://www.apache.org/licenses/LICENSE-2.0
+
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+"""Module to create bq table from the extracted ddl"""
+
+
+import sys
+import argparse
+import json
+import ast
+import datetime
+from google.cloud import bigquery
+from google.api_core.exceptions import NotFound
+from google.cloud import storage
+
+
+def create_log_table(project_id, target_dataset, client):
```

Comment:
It seems to me that the same function is in `snowflake_ddl_extraction.py`. Could this function be extracted to a common py file to be leveraged by both script without duplicating code?